src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo__header">
+      <a class="header__img" href="http://#"
+      target="_blank">
+        <img src="images/logo.png" width="40"
+        height="40" alt="moyo-logo">
+      </a>
+      <nav class="moyo__nav">
+        <ul class="nav__list">
+          <li>
+            <a class="nav__link is-active" href="http://#"
+            target="_blank">Apple</a>
+          </li>
+          <li>
+            <a class="nav__link" href="http://#"
+            target="_blank">Samsung</a>
+          </li>
+          <li>
+            <a class="nav__link" href="http://#"
+            target="_blank">Smartphones</a>
+          </li>
+
+          <li>
+            <a class="nav__link" data-qa="hover"
+            href="http://#" target="_blank">laptops & computers</a>
+          </li>
+
+          <li>
+            <a class="nav__link" href="http://#"
+            target="_blank">Gadgets</a>
+          </li>
+          <li>
+            <a class="nav__link" href="http://#"
+            target="_blank">Tablets</a>
+          </li>
+          <li>
```

Comment:
Check out [this list](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md), you need to format your code correctly for readablity, also don't forget about spaces between multiline siblings.
```suggestion
          <li>
            <a class="nav__link" href="#" target="_blank">
              Gadgets
            </a>
          </li>

          <li>
            <a class="nav__link" href="#" target="_blank">
              Tablets
            </a>
          </li>

          <li>
```

Reply:
Use "#" instead of "http://#". No need to have http part there.